main.tf

```diff
@@ -0,0 +1,191 @@
+*/
+
+# Terraform
+#--------------------------------------------------------------
+terraform {
+  required_version = ">= 0.10.2"
+}
+
+# Data
+#--------------------------------------------------------------
+data "aws_vpc" "default" {
+  id = "${var.vpc_id}"
+}
+
+data "aws_subnet_ids" "default" {
+  vpc_id = "${var.vpc_id}"
+}
+
+data "aws_iam_policy_document" "assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    effect  = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "default" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    effect = "Allow"
+
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "es:ESHttpGet",
+      "es:ESHttpPut",
+      "es:ESHttpPost",
+      "es:ESHttpHead",
+      "es:ESHttpDelete",
+      "es:Describe*",
+      "es:List*",
+    ]
+
+    effect = "Allow"
+
+    resources = [
+      "${var.es_domain_arn}",
+    ]
+  }
+}
+
+# Modules
+#--------------------------------------------------------------
+module "label" {
+  source     = "git::https://github.com/cloudposse/terraform-terraform-label.git?ref=tags/0.1.6"
+  namespace  = "${var.namespace}"
+  name       = "${var.name}"
+  stage      = "${var.stage}"
+  delimiter  = "${var.delimiter}"
+  attributes = "${var.attributes}"
+  tags       = "${var.tags}"
+  enabled    = "true"
+}
+
+# Locals
+#--------------------------------------------------------------
+locals {
+  lambda_path   = "${path.module}/elasticsearch_cleanup.zip"
+  function_name = "${module.label.id}-elasticsearch-cleanup"
+  enabled       = "${var.enabled == "true" ? true : false }"
+}
+
+# Resources
+#--------------------------------------------------------------
+resource "aws_lambda_function" "default" {
+  count            = "${local.enabled == true ? 1 : 0}"
+  filename         = "${local.lambda_path}"
+  function_name    = "${local.function_name}"
+  description      = "${module.label.id}-elasticsearch-cleanup"
+  timeout          = "${var.timeout}"
+  runtime          = "python${var.python_version}"
+  role             = "${aws_iam_role.default.arn}"
+  handler          = "es-cleanup.lambda_handler"
+  source_code_hash = "${base64sha256(file(local.lambda_path))}"
+  tags             = "${module.label.tags}"
+
+  environment {
+    variables = {
+      es_endpoint  = "${var.es_endpoint}"
+      index        = "${var.index}"
+      delete_after = "${var.delete_after}"
+      index_format = "${var.index_format}"
+      sns_alert    = "${var.sns_alert}"
+    }
+  }
+
+  vpc_config {
+    subnet_ids         = ["${data.aws_subnet_ids.default.ids}"]
+    security_group_ids = ["${aws_security_group.default.id}"]
+  }
+}
+
+resource "aws_security_group" "default" {
+  count       = "${local.enabled == true ? 1 : 0}"
+  name        = "${module.label.id}-elasticsearch-cleanup"
```

Comment:
We are not consistent about this, but technically should be using a new label with `attributes = ["elasticsearch", "cleanup"]`

Reply:
Otherwise, we ignore the user's delimiter selection